pkg/cmd/registry/rule/rulecmdutil/error.go

```diff
@@ -0,0 +1,30 @@
+package rulecmdutil
+
+import (
+	"github.com/redhat-developer/app-services-cli/pkg/core/localize"
+)
+
+type RegistryRuleError struct {
+	Localizer  localize.Localizer
+	InstanceID string
+}
+
+func (r *RegistryRuleError) ConflictError(ruleType string) error {
```

Comment:
Amazing job! This looks like generic handling for the registry so we can keep it at the top level

Reply:
> This looks like generic handling for the registry so we can keep it at the top level later on.
These are specific to the rules command.

> What I think we should do is to pull the switch logic into this file
Different rule operations can have different messages for same status code. It is better to have switch statements in command files.

Reply:
Ignore the comment above, I will be using [TransformInstanceError](https://github.com/redhat-developer/app-services-cli/blob/main/pkg/cmd/registry/registrycmdutil/error.go#L36).